### PR TITLE
MODE-2626 Fixes the custom binary storage integration with the WF kit

### PR DESCRIPTION
**Pull request by hchiorean**

### CODE REVIEW

File: deploy/jbossas/modeshape-jbossas-subsystem/src/main/java/org/modeshape/jboss/subsystem/AddCustomBinaryStorage.java [hunk 1]
```diff
@@ -39,17 +45,28 @@ protected void writeBinaryStorageConfiguration( String repositoryName,
...
+    
+    private void writeCustomProperty(EditableDocument binaries, Property property) {
+        ModelNode value = property.getValue();
+        if (!value.isDefined()) {
+            return;
         }
+        String name = property.getName();
+        if (COMMON_BINARY_ATTRIBUTES.contains(name)) {
+            // these are set via the base class...
+            return;
```
Comment (rhauch) [inline deploy/jbossas/modeshape-jbossas-subsystem/src/main/java/org/modeshape/jboss/subsystem/AddCustomBinaryStorage.java]: How does this work to set them via the base class? This method doesn't call 'super'.

Comment (hchiorean) [inline deploy/jbossas/modeshape-jbossas-subsystem/src/main/java/org/modeshape/jboss/subsystem/AddCustomBinaryStorage.java]: via [this method|https://github.com/ModeShape/modeshape/blob/master/deploy/jbossas/modeshape-jbossas-subsystem/src/main/java/org/modeshape/jboss/subsystem/AbstractAddBinaryStorage.java#L101] called first when initializing any binary store.

Comment (rhauch) [inline deploy/jbossas/modeshape-jbossas-subsystem/src/main/java/org/modeshape/jboss/subsystem/AddCustomBinaryStorage.java]: Thanks!

**Comment by rhauch** [comment]
@hchiorean, looks great.

**Pull request merged**